### PR TITLE
Added lab1 tasks

### PR DESCRIPTION
**Pull request by RobertProdanov03**

### CODE REVIEW

File: LabWork1_RobertProdanov/LabWork_RobertProdanov.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*Task1*/
+
+use employees_test_robertprodanov
+
```
Comment (sslavova) [inline LabWork1_RobertProdanov/LabWork_RobertProdanov.sql:4]: salary and previous experience can be nullable.

File: LabWork1_RobertProdanov/LabWork_RobertProdanov.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+alter table employees add salary decimal(10,2) not null;
+alter table employees add previous_experience enum('yes','no') not null;
+
+/*Task2*/
+
```
Comment (sslavova) [inline LabWork1_RobertProdanov/LabWork_RobertProdanov.sql:9]: Manager ID and location can be nullable.

**Review by sslavova — CHANGES_REQUESTED** [review]

File: LabWork1_RobertProdanov/LabWork_RobertProdanov.sql (hunk 1 above)
Comment (RobertProdanov03) [reply LabWork1_RobertProdanov/LabWork_RobertProdanov.sql:4]: I fixed the problem

**Comment by RobertProdanov03** [comment]
I fixed the problem

**Pull request merged**